vms/platformvm/vm_test.go

```diff
@@ -1675,7 +1675,7 @@ func TestUnverifiedParent(t *testing.T) {
 			},
 		}},
 	}}
-	require.NoError(tx1.Initialize(txs.Codec))
+	require.NoError(tx2.Initialize(txs.Codec))
```

Comment:
I think this was a bug previously that wasn't caught since it was initialized when we create the block later